scripts/README.md

```diff
@@ -1,16 +1,29 @@
-# kibana dev scripts
+# Kibana Dev Scripts
 
 This directory contains scripts useful for interacting with Kibana tools in development. Use the node executable and `--help` flag to learn about how they work:
 
 ```sh
 node scripts/{{script name}} --help
 ```
 
-## for developers
+## For Developers
 
 This directory is excluded from the build and tools within it should help users discover their capabilities. Each script in this directory must:
 
 - require `src/babel-register` to bootstrap babel
-- call out to source code that is in the `src` directory
+- call out to source code in the [`src`](../src) or [`packages`](../packages) directories
 - react to the `--help` flag
-- run everywhere OR check and fail fast when a required OS or toolchain is not available
\ No newline at end of file
+- run everywhere OR check and fail fast when a required OS or toolchain is not available
+
+## Functional Test Scripts
+
+### `node scripts/functional_tests`
+Runs all the functional tests: selenium tests and api integration tests. Uses the [@kbn/test](../packages/kbn-test) library to run Elasticsearch and Kibana servers and tests against those servers, for multiple server+test setups. In particular, calls out to [`runTests()`](../packages/kbn-test/src/functional_tests/tasks.js#L31-L49)
+
```

Comment:
__nit:__
> (../packages/kbn-test/src/functional_tests/tasks.js#L31-L49)

Maybe we shouldn't be that specific, line numbers can easily get outdated and nobody will notice.

Reply:
sure, i can remove the line numbers :)